rotkehlchen/tests/utils/constants.py

```diff
@@ -80,3 +81,35 @@
     endpoint='https://mainnet.eth.cloud.ava.do/',
     owned=False,
 )
+
+TX_HASHES_PAIRS = {
```

Comment:
Heh that's not really what I meant before. I meant to just put the few repeated deserialized hashes that repeat in a variable and reuse it.

This is a bit of an overkill.

And if anything the name is wrong and confusing. It should be `TXHASH_HEX_TO_BYTES`.

I am not sure I like this approach though. But if you really wanna use it then rename.